### PR TITLE
Update guides.md for system_ext

### PR DESCRIPTION
**Pull request by osm0sis**

### CODE REVIEW

File: docs/guides.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ A Magisk module is a folder placed in `/data/adb/modules` with the structure bel
 │   │
 │   ├── vendor              <--- A symlink to $MODID/system/vendor
 │   ├── product             <--- A symlink to $MODID/system/product
+│   ├── system_ext          <--- A symlink to $MODID/system/system_ext
```
Comment (pollon23) [inline docs/guides.md:55]:
```suggestion
│   ├── system_ext          <--- A symlink to $MODID/system/system_ext
```

**Review by pollon23 — COMMENTED** [review]
``

**Review by pollon23 — APPROVED** [review]
``

File: docs/guides.md (hunk 1 above)
Comment (pollon23) [reply docs/guides.md:55]: [](url)

Comment (osm0sis) [reply docs/guides.md:55]: What the fuck?

**Review by topjohnwu — APPROVED** [review]

**Pull request merged**